.github/workflows/ci.yml

```diff
@@ -81,13 +80,11 @@ jobs:
                 apk add cmake make musl-dev perl && \
                 cargo run -p package -- --dest-dir dist/ scie-pants
               '
-          cargo run -p package -- --dest-dir dist/ scie \
-            --scie-pants dist/scie-pants --tools-pex dist/tools.pex
           echo
           echo "Running under: $(uname -a)"
           echo
           PANTS_BOOTSTRAP_GITHUB_API_BEARER_TOKEN=${{ secrets.GITHUB_TOKEN }} \
             cargo run -p package -- test \
-              --tools-pex dist/tools.pex --scie-pants dist/scie-pants-linux-x86_64 \
+              --tools-pex dist/tools.pex --scie-pants dist/scie-pants \
```

Comment:
So this is the sort of thing a review eye might gloss over, but there is a small universe hidden here:
1. The default command of the scie-pants scie is to run the embedded scie-pants Rust binary: https://github.com/pantsbuild/scie-pants/blob/14ed78482a6e39d7f01a2d01e1b169c862c104ba/package/scie-pants.toml#L48-L51
2. On the LHS of this diff, the scie-pants scie was passed off as `--scie-pants` instead of the scie-pants Rust binary!

This mostly works because of 1 above except for env var manipulation by the enveloping scie. It was this double-manipulation of env vars (the outer scie-pants scie manipulates env vars and runs the default command which turns out not to be the scie-pants Rust binary but another scie-pants scie!) that led to a PANTS_BIN_NAME test failure.